envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadataTest.kt

```diff
@@ -318,4 +333,42 @@ class NodeMetadataTest {
         // then
         assertThat(duration).isNull()
     }
+
+    @ParameterizedTest
+    @MethodSource("validStatusCodeFilterData")
+    fun `should set statusCodeFilter for accessLogFilter`(input: String, op: ComparisonFilter.Op, code: Int) {
+        // given
+        val proto = accessLogFilterProto(statusCodeFilter = input)
+
+        // when
+        val incoming = proto.structValue?.fieldsMap?.get("status_code_filter").toStatusCodeFilter(
+            AccessLogFilterFactory()
+        )
+
+        // expects
+        assertThat(incoming?.comparisonCode).isEqualTo(code)
+        assertThat(incoming?.comparisonOperator).isEqualTo(op)
+    }
+
+    @ParameterizedTest
+    @CsvSource(
+        "LT:123",
+        "equal:400",
+        "eq:24",
+        "GT:200"
+    )
+    fun `should not set statusCodeFilter for accessLogFilter`(input: String) {
```

Comment:
In the previous test there is MethodSource: `validStatusCodeFilterData` -> that way I know why it should set statusCodeFilter, in this method you used @CvsSource and I do not know why it should not be set. I would clarify in the test name (something like: `should not set statusCodeFilter for accessLogFilter for invalid status code filter data`) or use MethodSource like before.

Reply:
changed test name